github/messages_test.go

```diff
@@ -82,6 +83,54 @@ func TestValidatePayload(t *testing.T) {
 	}
 }
 
+func TestValidateFormPayloadGET(t *testing.T) {
```

Comment:
I looked at adding this test to the original `TestValidatePayload` test cases but thought the additional complexity of adding it to that test was too much since you need to change how the test is constructing the request.